diagnostics/app/controllers/DiagnosticsController.scala

```diff
@@ -40,9 +40,17 @@ object DiagnosticsController extends Controller with Logging {
     TinyResponse.noContent()
   }
 
-  def cssOptions = postOptions
+  def csp = Action(jsonParser) { implicit request =>
+    val shouldReport = (new Instant().getMillis) % 10000 == 1
```

Comment:
`Unless some pages are more likely to be hit in a certain millisecond than others?` 
That's what I am questioning 😁 

Also a quick test in scala repl shows `scala.util.Random` is much faster than `Instant().getMillis`
```
scala> def time[R](block: => R): R = {
     | val t0 = System.nanoTime()
     | val result = block
     | println("Elapsed time: " + (System.nanoTime - t0) / 1e6 + "ms")
     | result
     | }
time: [R](block: => R)R

scala> time {0.to(1000000000).foreach(ignored => (new Instant().getMillis) % 10000 )}
Elapsed time: 61791.900968ms

scala> time {0.to(1000000000).foreach(ignored => scala.util.Random.nextInt(10000) )}
Elapsed time: 24994.601662ms
```